### PR TITLE
Angular: Adds back two angular directives that are still used by remaining angular bits and plugins

### PR DESCRIPTION
**Pull request by torkelo**
Adds back bs-tooltip and bs-typeahead that where removed when I removed the angular-strap file

### CODE REVIEW

**Review by alexanderzobnin — APPROVED** [review]
LGTM!

File: public/app/plugins/datasource/mssql/partials/config.html [hunk 1]
```diff
@@ -1,80 +1,83 @@
-
 <h3 class="page-heading">MS SQL connection</h3>
 
 <div class="gf-form-group">
-	<div class="gf-form max-width-30">
-		<span class="gf-form-label width-7">Host</span>
-		<input type="text" class="gf-form-input" style="width: 352px" ng-model='ctrl.current.url' placeholder="localhost" bs-typeahead="{{['localhost', 'localhost" required></input>
```
Comment (torkelo) [inline public/app/plugins/datasource/mssql/partials/config.html:7]: this line is the only one that changed, the bs-typeahead line was an incomplete expression. Then prettier formatted the whole document :)

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: public/app/angular/bsTypeahead.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            if (controller) {
+              scope.$apply(function () {
+                controller.$setViewValue(value);
+              });
+            }
+            scope.$emit('typeahead-updated', value);
+            return value;
+          },
+        });
+        var typeahead = element.dat('typeahead');
```
Comment (alexanderzobnin) [inline public/app/angular/bsTypeahead.ts:38]: Should be 
```ts
var tooltip = element.data('tooltip');
```

**Pull request merged**